templates/layout/page.html.twig

```diff
@@ -93,17 +93,64 @@
     {{ page.content_top }}
   {% endif %}
 
-  {% if not node %}
-    <div class="lgd-container">
+  {% if node %}
```

Comment:
Any particular reason why we are only rendering this section if we are on a node page? I think this is going to mean taxonomy pages, views pages, etc are not going to render anything in here (for example, sidebars).

The reason I had the "if not node" clause, was so that _all_ pages except node pages would default to the site's default width (sensible default) and then the width of items in nodes would be set by the node template (with node--full.html.twig) have a sensible default (but means something like node--landing-page.html.twig could be 100% width for some fields.

Reply:
I largely copied the sidebar stuff from localgov_base. I re-arranged the logic slightly to fit, it should print everything, but happy to take suggestions if you'd like it done differently.

Currently the logic is:
```twig
  {% if node %}
    {% if has_sidebars %}
      {# Handle node content with sidebars #}
    {% else %}
      {# Just print node content  #}
    {% endif %}
  {% else %}
    {% Print none-node content without sidebars %}
  {% endif %}
```

Reply:
The issue is that we are only going to ever render sidebars on node pages if we follow this logic. I'm happy with the part copied from localgov_base, but I think we need a different `if` statement around it, perhaps the one we had there already.